samples/53.teams-messaging-extensions-action-preview/src/main/java/com/microsoft/bot/sample/teamsactionpreview/TeamsMessagingExtensionsActionPreviewBot.java

```diff
@@ -50,163 +41,120 @@ public TeamsMessagingExtensionsActionPreviewBot(Configuration configuration) {
     }
```

Comment:
I don't think the Configuration (or the constructor) is needed at all for this bot.  The private members appId and appPassword, along with the constructor can just be deleted.